### DISCUSSION TITLE
请问readme里的obsidian主题叫什么？

### DISCUSSION THREAD

**Discussion by nnysldrwv** [Quorafind/Obsidian-Thino#238] [Q&A 问答]
![Memos-Desktop](https://user-images.githubusercontent.com/31592021/177560954-d29e5d3a-704e-49ed-89a1-4c309c12581e.png)

请问这个主题是什么？

**Comment by ycbrd** [comment]
感觉仿佛就是blue topaz

**Comment by juestchaos** [comment]
两个截图，都是用去年的 bluetpoz 主题截图的
热力图的样式，可以参考 pkmer 教程自定义颜色
